bin/common.h

```diff
@@ -74,6 +98,8 @@ struct conn_settings {
     int max_conns;
     const char *ca_dir;
     const char *ca_file;
+    const char *serialize_out;
+    const char *deserialize_in;
```

Comment:
Nit: I know these scripts don't follow any of the rules, but
```suggestion
    const char *serialize_out = NULL;
    const char *deserialize_in = NULL;
```

Reply:
It's a struct field, you can't assign a value to it.